pattern_import_export/models/common.py

```diff
@@ -0,0 +1,5 @@
+# Copyright 2020 Akretion France (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+IDENTIFIER_SUFFIX = "/key"
```

Comment:
The key already exist here : https://github.com/akretion/pattern-import-export/blob/12.0-refactor/pattern_import_export/models/ir_fields.py#L10